src/pytest/test.py

```diff
@@ -1021,22 +1011,29 @@ def testGeoDeletion(env):
             'g1', "-0.1757,51.5156",
             'g2', "-0.1757,51.5156",
             't1', "hello")
+    env.cmd('ft.add', 'idx', 'doc3', 1.0, 'fields',
+            'g1', "-0.1757,51.5156",
+            't1', "hello")
```

Comment:
Any reason to add another document here? what is it testing that was not tested before?

Reply:
See below